pangeo-notebook/environment.yml

```diff
@@ -112,7 +112,8 @@ dependencies:
  - xarray_leaflet
  - xarray-spatial
  - xbatcher
- - xcape
+ # https://github.com/conda-forge/xcape-feedstock/pull/22
+ #- xcape
```

Comment:
dropping b/c blocking both #396 and #497 and doesn't seem very active. people can always install it on top of the default environment